### ISSUE TITLE
In-text citations have comma separator between author year

### ISSUE THREAD

**Issue by brentonsmith** [chewxy/SDIPUTS#2]
UTS-Harvard does not use a comma between author and year in the in-text citation.

With natbib this is easily solved with "\setcitestyle{aysep={}}", can it be solved in the .bst file however?
Can probably add that macro directly  in the preamble witht he bst file somewhere, but that might require a dependency on natbib I guess which im not sure you currently have?

Alternative is a "quick start guide" in the read me which shows that macro + natbib being used.

**Comment by chewxy** [comment]
That was my solution as well

**Comment by brentonsmith** [comment]
I've also noticed in-text citations with multiple authors are using "and" rather than "&" to separate authors, i don't how to fix this with setcitestyle or in the bst file.

See http://www.lib.uts.edu.au/help/referencing/harvard-uts-referencing-guide/journal-article#authors

Modifying the bbl.and file seems a bit too excessive and will probably break other things, what do you think?

**Comment by chewxy** [comment]
Gah... I spent so much time on making `and` work in in-text citations... (the sheets I got from tutes had "and" instead of "&")

I think it's best to switch out the `bbl.and` (which is what I eventually settled on)

**Comment by chewxy** [comment]
Actually, no... there is a bunch of things in `{format.names}` that uses `"\harvardand{}`

**Comment by brentonsmith** [comment]
Yeah, unfortunately there are a few issues here and there on the website and inconsistencies across certain parts or the different guides.

UTS-Library have told me the website (not the PDF) is most up to date and should be considered reference source. But I think EndNote file may even be a better reference source, apparently it handles boutique references better than the .csl file (Zotero, but UTS style was built for Mendeley).

**Comment by chewxy** [comment]
(╯°□°）╯︵ ┻━┻

**Comment by chewxy** [comment]
Using `&` in-text looks weird outside of parens

**Comment by chewxy** [comment]
"Foo  & Barr (1900) said baz`

**Comment by brentonsmith** [comment]
Oh, I more meant "Baz (Foo & Bar 1900)"

Inline in-text quotes appear to use "and", see http://www.lib.uts.edu.au/help/referencing/harvard-uts-referencing-guide/more-information#in-text
